js/positions.js

```diff
@@ -69,9 +69,9 @@ const pos2 = [
 const pos3 = [
   "48%,2%",
   "48%,10%",
-  "10%,20%",
-  "48%,20%",
-  "85%,20%",
+  "10%,19%",
+  "48%,19%",
+  "85%,19%",
```

Comment:
Hvorfor har du ændret det?

Reply:
Der var også et problem med plads for initials
![billede](https://user-images.githubusercontent.com/82437169/170240468-b9b4472b-6f65-4cc4-bf53-a692a61788f0.png)

Reply:
Nope, det er der ikke, der var en opgave på det som christoffer har lavet

Reply:
også det som er problemet med din merge conflict.

Reply:
> Nope, det er der ikke, der var en opgave på det som christoffer har lavet

Kan ikke helt forstå overstående

Reply:
Det du ændre her har ikke noget med din opgave at gøre.
Der ligger en opgave med præcis det fix her, og det er lavet.
https://kea-gx.atlassian.net/jira/software/projects/GXH/boards/7?selectedIssue=GXH-32

Reply:
så du skal ændre det tilbare til 1.2ch igen

Reply:
Ups..Ok, jeg laver det hele om i css igen